src/Kiota.Builder/CodeDOM/CodeMethod.cs

```diff
@@ -32,13 +33,21 @@ public enum HttpMethod {
 
     public class CodeMethod : CodeTerminal, ICloneable, IDocumentedElement
     {
-        public CodeMethod(CodeElement parent): base(parent) {}
+        public CodeMethod(): base() {}
```

Comment:
We could probably just remove this as the `CodeTerminal` base is abstract and no longer provides a custom constructor to inherit.

Reply:
Same for `CodeParameter` class

Reply:
thanks, fixed via cf78a22 (pending push)